src/diffusers/loaders/lora.py

```diff
@@ -391,6 +391,10 @@ def load_lora_into_unet(
         # their prefixes.
         keys = list(state_dict.keys())
 
+        if all(key.startswith("unet.unet") for key in keys):
+            deprecation_message = "Keys starting with 'unet.unet' are deprecated."
+            deprecate("unet.unet keys", "0.25", deprecation_message)
```

Comment:
```suggestion
            deprecate("unet.unet keys", "0.27", deprecation_message)
```

We're already at 0.25